homeassistant/components/hue/bridge.py

```diff
@@ -97,8 +97,13 @@ def host(self):
 
         # If setup was successful, we set api variable, forwarded entry and
         # register service
-        return await self.hass.config_entries.async_forward_entry_unload(
-            self.config_entry, 'light')
+        light = (await self.hass.config_entries.async_forward_entry_unload(
```

Comment:
Use `asyncio.gather` to await all coroutines together and gather the result in a list.